spec/models/equipment_model_spec.rb

```diff
@@ -151,6 +151,59 @@
     end
   end
 
+  describe 'counter cache' do
+    before(:each) do
+      @model = FactoryGirl.create(:equipment_model)
+      @res = FactoryGirl.create(:checked_out_reservation,
+                                equipment_model: @model,
+                                start_date: Time.zone.today - 2.days,
+                                due_date: Time.zone.today - 1.day)
+      # have to do this manually because update_overdue marks the res as overdue
+      @res.update_columns(overdue: false)
+      @model.decrement(:overdue_count)
+    end
+    it 'increments when a reservation is marked as overdue' do
+      @res.update_attribute(:overdue, true)
+      expect(@model.overdue_count).to eq(1)
+    end
+    it 'only increments once per reservation' do
+      @res.update_attribute(:overdue, true)
+      expect { @res.update_attribute(:notes, 'test') }.not_to(
```

Comment:
I think that it makes sense for the multi-line ones (I guess I could just use `\`? Is that preferable?), but I'll update the others